### PR TITLE
Chore/ccp 263

### PR DESCRIPTION
**Pull request by sri-9597**
# Guidelines for Pull Requests

If you haven't yet read our code review guidelines, please do so, You can find them [here](https://diging.atlassian.net/wiki/spaces/DIGING/pages/2256076801/Code+Review+Guidelines).

Please confirm the following by adding an x for each item (turn `[ ]` into `[x]`).

- [x] I have removed all code style changes that are not necessary (e.g. changing blanks across the whole file that don’t need to be changed, adding empty lines in parts other than your own code)
- [x] I am not making any changes to files that don’t have any effect (e.g. imports added that don’t need to be added)
- [x] I do not have any sysout statements in my code or commented out code that isn’t needed anymore
- [x] I am not reformatting any files in the wrong format or without cause. 
- [x] I am not changing file encoding or line endings to something else than UTF-8, LF
- [x] My pull request does not show an insane amount of files being changed although my ticket only requires a few files being changed
- [x] I have added Javadoc/documentation where appropriate
- [x] I have added test cases where appropriate
- [x] I have explained any part of my code/implementation decisions that is not be self-explanatory

## Please provide a brief description of your ticket 
Update dependencies
  
## Anything else the reviewer needs to know?

... describe here ...

### CODE REVIEW

File: Conceptpower+Spring/src/main/webapp/WEB-INF/spring/spring-security.xml [hunk 1]
```diff
@@ -15,19 +15,25 @@
...
-		<intercept-url pattern="/welcome*" access="permitAll" />
-
-		<intercept-url pattern="/resources/**" access="permitAll" />
+		  
+		<intercept-url pattern="/conceptpower/rest/concept/add" access="hasRole('ROLE_CP_ADMIN')" />
 
 		<intercept-url pattern="/auth/user/**" access="hasRole('ROLE_CP_ADMIN')" />
 		<intercept-url pattern="/auth/index/**" access="hasRole('ROLE_CP_ADMIN')" />
+		<intercept-url pattern="/auth/conceptlist/**" access="hasRole('ROLE_CP_ADMIN')" />
+		<intercept-url pattern="/auth/concepttype/**" access="hasRole('ROLE_CP_ADMIN')" />
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/webapp/WEB-INF/spring/spring-security.xml]: Those two should be available to every user, not just admins.

File: Conceptpower+Spring/src/main/webapp/WEB-INF/spring/spring-security.xml [hunk 2]
```diff
@@ -15,19 +15,25 @@
 		<logout
 		  logout-success-url="/"
 		  logout-url="/signout" delete-cookies="JSESSIONID" />
-		
-		<intercept-url pattern="/welcome*" access="permitAll" />
-
-		<intercept-url pattern="/resources/**" access="permitAll" />
+		  
+		<intercept-url pattern="/conceptpower/rest/concept/add" access="hasRole('ROLE_CP_ADMIN')" />
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/webapp/WEB-INF/spring/spring-security.xml]: Should be available to any user not just admins.

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
Make it so, Jenkins.

File: Conceptpower+Spring/src/main/webapp/WEB-INF/spring/spring-security.xml [hunk 3]
```diff
@@ -15,19 +15,25 @@
...
-		<intercept-url pattern="/welcome*" access="permitAll" />
-
-		<intercept-url pattern="/resources/**" access="permitAll" />
+		  
+		<intercept-url pattern="/conceptpower/rest/concept/add" access="isAuthenticated()" />
 
 		<intercept-url pattern="/auth/user/**" access="hasRole('ROLE_CP_ADMIN')" />
 		<intercept-url pattern="/auth/index/**" access="hasRole('ROLE_CP_ADMIN')" />
+		<intercept-url pattern="/auth/conceptlist/**" access="isAuthenticated()" />
+		<intercept-url pattern="/auth/concepttype/**" access="isAuthenticated()" />
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/webapp/WEB-INF/spring/spring-security.xml]: are those two actually neccessary? aren't they covered by line 26?

Comment (sri-9597) [reply Conceptpower+Spring/src/main/webapp/WEB-INF/spring/spring-security.xml]: Oh yes, sorry about that. Fixed this and I also rearranged the intercept-urls access-level wise

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
Jenkins, run integration tests please

**Pull request merged**